homework/css/components/footer.css

```diff
@@ -0,0 +1,34 @@
+footer {
+  grid-area: footer;
+  background-color:  rgba(67, 71, 67, 0.7);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 30px;
+}
+
+footer .media-icons {
+  display: flex;
+  padding: 0 30px;
+}
+
+footer .media-icons div {
+  cursor: pointer;
+}
+
+footer .media-icons span {
+  font-size: 1.5em;
+  display: inline-block;
+  margin-left: 40px;
```

Comment:
Better don't set margin-left to the first element, as it will cause additional issues on responsiveness adjustments. In this project it is ok. But in general, try to avoid such situations

Reply:
Done